pxtlib/emitter/cloud.ts

```diff
@@ -92,21 +92,22 @@ namespace pxt.Cloud {
     // 1h check on markdown content
     const MARKDOWN_EXPIRATION = 1 * 60 * 60 * 1000;
     export function markdownAsync(docid: string, locale?: string, live?: boolean): Promise<string> {
+        const branch = pxt.appTarget.versions && pxt.appTarget.versions.target || '?';
```

Comment:
Markdown is not versioned by targetVersion - it's independent, so I'm not sure this helps much

Reply:
Removed.